### PR TITLE
[INLONG-8231][Manager] Add tenant into param and body of each request

### LINKED ISSUE #8231: [Feature][Manager] Add tenant into param and body of each request
### Description

Parent #7914 
Add tenant into param and body of each request

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by vernedeng**
### Prepare a Pull Request
*(Change the title refer to the following example)*

- Title Example: [INLONG-XYZ][Component] Title of the pull request

*(The following *XYZ* should be replaced by the actual [GitHub Issue](https://github.com/apache/inlong/issues) number)*

- Fixes #8231
- Parent #7914 

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve?*

### Modifications

*Describe the modifications you've done.*

### Verifying this change

*(Please pick either of the following options)*

- [ ] This change is a trivial rework/code cleanup without any test coverage.

- [ ] This change is already covered by existing tests, such as:
  *(please describe tests)*

- [ ] This change added tests and can be verified as follows:

  *(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a follow-up issue for adding the documentation

### CODE REVIEW

File: inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/utils/HttpContextUtils.java [hunk 1]
```diff
@@ -104,7 +104,10 @@ public static String getBodyString(ServletRequest request) {
                 }
             }
         } catch (IOException e) {
-            log.error("failed to get body string of request={}", request);
+            log.error("failed to get body string of request={}", request, e);
+        }
+        if (sb.length() == 0) {
```
Comment (healchow) [inline inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/utils/HttpContextUtils.java:109]: 1. `sb` -> `builder`
2. Please extract a constant for `{}`, such as `EMPTY_BODY`.

File: inlong-manager/manager-web/src/test/java/org/apache/inlong/manager/web/filter/TenantInsertionFilterTest.java [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import java.io.IOException;
+import java.util.Map;
+
+public class TenantInsertionFilterTest extends WebFilterConfig {
```
Comment (healchow) [inline inlong-manager/manager-web/src/test/java/org/apache/inlong/manager/web/filter/TenantInsertionFilterTest.java:42]: Please add a javadoc for this test class, such as `Test for {@link TenantInsertionFilter}`

**Review by fuweng11 — APPROVED** [review]

File: inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/utils/HttpContextUtils.java (hunk 1 above)
Comment (vernedeng) [reply inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/utils/HttpContextUtils.java]: fixed, thx

File: inlong-manager/manager-web/src/test/java/org/apache/inlong/manager/web/filter/TenantInsertionFilterTest.java (hunk 2 above)
Comment (vernedeng) [reply inlong-manager/manager-web/src/test/java/org/apache/inlong/manager/web/filter/TenantInsertionFilterTest.java:45]: fixed, thx

**Review by dockerzhang — APPROVED** [review]

**Review by healchow — APPROVED** [review]

**Pull request merged**